### ISSUE TITLE
Add paging capabilities to GeoServer (and primary keys to database?)

### ISSUE THREAD

**Issue by thareUSGS** [USGS-Astrogeology/PDS-Pipelines#530]
from: http://osgeo-org.1560.x6.nabble.com/Anybody-to-confirm-this-WFS-problem-tp5381615p5389186.html

```
GeoServer has needed primary keys to support paging since 10 years ago, when
this ticket got worked on ...:
https://osgeo-org.atlassian.net/browse/GEOT-1794
https://github.com/geotools/geotools/wiki/feature-paging-and-query-capabilities

Setting up a WFS server backed by a table without primary keys is problematic for other reasons too,
for example, features need to be identifiable via id.

Long story short, it would still be a good idea for someone to contact the server administrator and
let them know they should rectify/improve their setup.
```
